docs/welcome-to-payid.md

```diff
@@ -1,17 +1,19 @@
 ---
-id: payid-overview
-title: Overview
-sidebar_label: Overview
+id: welcome-to-payid
+title: Welcome to PayID
+sidebar_label: Welcome to PayID
```

Comment:
Food for thought: will this page need any revision for the new docs landing page?